.lagoon.yml

```diff
@@ -39,6 +39,8 @@ tasks:
         command: es-curl PUT _cluster/settings -d '{"persistent":{"cluster.routing.allocation.enable":null}}'
         service: logs-db
 
+production-backup-retention:
+  monthly: 2
```

Comment:
do we need to define this here? we're not going to use lagoon to deploy lagoon in the future, so I think this can go?

Reply:
I added this both for testing purposes as well as to provide a bit of context on how to use this functionality. It shouldn't hurt our testing setup to keep it in. :)

Reply:
we changed this now to be `backup-retention.production`

> added this both for testing purposes
mhh testing should happen before the production deployment of Lagoon and this is only is used when we deploy lagoon to production. If we realize then that this is broken, then it's too late ;)

> how to use this functionality.
Lagoon documentation happens in the lagoon docs and not in `.lagoon.yml` of Lagoon itself.
As we have proper documentation in the docs already, let's remove this here